tests/components/matter/test_sensor.py

```diff
@@ -250,6 +260,24 @@ async def test_battery_sensor(
     assert entry
     assert entry.entity_category == EntityCategory.DIAGNOSTIC
 
+    """Test battery voltage sensor."""
```

Comment:
Define a new test function for that uses the fixture `eve_contact_sensor_node` fixture that is already available.